addons/account/migrations/12.0.1.1/post-migration.py

```diff
@@ -220,6 +220,42 @@ def set_default_taxes(env):
         })
 
 
+def fill_res_company_invoice_reference_type(cr):
```

Comment:
Why this change here?

Reply:
Well, I am not really sure of this long query and the other long in the l10n_be_invoice_bba. I was waiting your opinion. I can remove them.

Reply:
But I want to know why is needed to be done in account. Rationale?

Reply:
Ah, because this part is `account` stuff. We didn't saw when we migrated `account`, the relation between `reference_type` and `invoice_reference_type`.

Reply:
Then it should be a different PR...

Reply:
done in https://github.com/OCA/OpenUpgrade/pull/1941.